tests/test_filter_classes.py

```diff
@@ -343,15 +343,15 @@ def test_filter_iir_process_complex(impulse_complex):
 
 def test_filter_iir_process_state(impulse):
     coeff = np.array([[1, 1/2, 0], [1, 0, 0]])
-    filt = fo.FilterIIR(coeff, impulse.sampling_rate, state=[0, 0])
+    filt = fo.FilterIIR(coeff, impulse.sampling_rate, state=[[[0, 0]]])
```

Comment:
Here and below, there was a warning because the filter state was provided in the wrong shape.